app/Services/VoucherEvaluator/EvaluatorFactory.php

```diff
@@ -71,23 +72,32 @@ private static function createStandardEvaluations(Carbon $offsetDate)
     {
         return $evaluations = [
             Child::class => [
+                'credits' => [
+                    new ChildIsUnderOne($offsetDate, 3),
```

Comment:
- One day this will be abstracted to pure data.
- The value may change in the near future

So i'm happy to be explicit here.

I do need a better solution for the `offsetDate`, which is very repetitive. This wont be the last time these rules get a jiggle though.